File: APIflow/python/tutorials/basicWorkflow.py
import sys, getopt

exec(open("../APIFlow.py").read())

email, password, apiKey = sys.argv[1], sys.argv[2], sys.argv[3]

sessionKey = auth(email, password, apiKey)

processTypes = listProcesstypes(sessionKey)

for d in processTypes:
	if(d["name"] == "demo"):
		procAck = startProcess(sessionKey, d["name"])
		process = getProcess(sessionKey, procAck["id"])
		for t in process["TASKS"]:
			res = completeTask(sessionKey, t["ID"])
			print(res)

File: APIflow/php/tutorials/basicWorkflow.php
<?php

include('../APIFlow.php');

$sessionKey = auth($argv[1], $argv[2], $argv[3]);
$processtypes = listProcesstypes($sessionKey);

foreach($processtypes as $t){
	if($t["name"] == "demo"){
		$demo = $t;

		$started = startProcess($sessionKey, $demo["name"]);
		$process = getProcess($sessionKey, $started["id"]);
		foreach($process["TASKS"] as $t){
			$res = completeTask($sessionKey, $t["ID"]);
			echo $res;
		}
	}
}

?>